tests/lean_spec/subspecs/containers/test_state.py

```diff
@@ -64,33 +69,42 @@ def genesis_state(sample_config: Config) -> State:
 
 
 def _create_block(
-    slot: int, parent_header: BlockHeader, votes: List[SignedVote] | None = None
-) -> SignedBlock:
+    slot: int,
+    parent_header: BlockHeader,
+    votes: Iterable[ValidatorAttestation] | None = None,
+) -> Block:
     """
-    Helper: construct a valid SignedBlock for a given slot.
+    Helper: construct a valid `Block` for a given slot.
 
-    Notes
-    -----
-    - Uses round-robin proposer selection with modulus 10 (aligned with sample_config).
-    - Sets state_root to zero; STF will compute and validate the real root.
-    - Accepts an optional list of SignedVote to embed in the body.
+        Notes
+        -----
+        - Uses round-robin proposer selection with modulus 10 (aligned with the
+            devnet configuration).
+        - Sets state_root to zero; STF will compute and validate the real root.
+        - Accepts an optional collection of validator attestations to embed in
+            the body.
 
     Parameters
     ----------
     slot : int
         Slot number for the new block.
     parent_header : BlockHeader
         The header of the parent block to link against.
-    votes : list[SignedVote] | None
+    votes : Iterable[ValidatorAttestation] | None
         Optional attestations to include.
 
     Returns
     -------
-    SignedBlock
-        A signed block wrapper containing the constructed block.
+    Block
+        The constructed block message with attestations embedded.
     """
+    normalized_attestations = []
+    if votes is not None:
+        for attestation in votes:
+            normalized_attestations.append(attestation)
```

Comment:
This seems strange to me. Why do we need to reconstruct the same list? Or I just don't know python enough.